build.override.targets

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="WcfSetup" Condition="'$(WithCategories)' == 'OuterLoop'" BeforeTargets="Test">
+    <Exec Command="$(MySetupScript) > $(MyLogFile)" />
+    <ReadLinesFromFile File="$(MyLogFile)" >
```

Comment:
Thanks for the clarification @StephenBonikowsky -- I won't code review the parts you expect to be replaced.